### DISCUSSION TITLE
REMOTE_CONTENT_ERROR: The remote server hosting the media failed to return valid data

### DISCUSSION THREAD

**Discussion by onosendi** [deepgram/community#721] [General help] [answered]
I'm getting the error given in the title with the code shown below. I'm sending the same URL (from our server) to deepgram, and it has about a 25% success rate. If I were to upload the audio to Dropbox, it has 100% success rate. Any ideas on what could be causing this issue? The URL hosted on our server *seems* to be fine.

Node version: v20.12.2
@deepgram/sdk version: 3.2.0

```javascript
const deepgram = createClient(config.deepgram.api_key);

async function send_deepgram_request(url) {
    const options = {
        model: 'nova-2-phonecall',
        summarize: 'v2',
        diarize: true,
        redact: 'pci',
        smart_format: true,
        numerals: true,
    };

    const { result, error } = await deepgram
        .listen
        .prerecorded
        .transcribeUrlCallback(
            { url },
            new CallbackUrl(config.deepgram.webhook),
            options,
        );

    return { result, error };
}
```

Here's the response, along with a `request_id`:

```
  {
    status: 'rejected',
    reason: Error: DeepgramApiError: {"err_code":"REMOTE_CONTENT_ERROR","err_msg":"The remote server hosting the media failed to return valid data.","request_id":"1f4ca608-0fa7-40eb-99fe-136374c2e32d"}
  },

```

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jpvajda** [comment, marked as answer]
@onosendi  I'm curious what type of file is this URL pointing to? Is it a public URL? If you check out this Docs page it might give you some ideas on troubleshooting this error: https://developers.deepgram.com/reference/errors#400-failure-to-fetch-remote-text-from-url
